be/src/exprs/in_const_predicate.hpp

```diff
@@ -326,6 +328,33 @@ class VectorizedInConstPredicate final : public Predicate {
         return evaluate_with_filter(context, ptr, nullptr);
     }
 
+    ColumnPtr get_all_values() const {
+        ColumnPtr values = ColumnHelper::create_column(TypeDescriptor{Type}, true);
+        if constexpr (isSliceLT<Type>) {
+            for (auto c : _string_values) {
+                values->append_datum(c->get(0));
```

Comment:
Whyn not use Column::append(const Column& src)?

Reply:
if we use append(const Column& src) we must make sure the dst column and src column are the same type, we must consider nullable/const...